previews/primer/alpha/action_list_preview.rb

```diff
@@ -459,6 +459,48 @@ def item_active
           component.with_item(label: "Active item", href: "/", active: true)
         end
       end
+
+      # @label Long label
+      #
+      # @param truncate_label [Symbol] select [none, truncate, show_tooltip]
+      # @snapshot interactive
```

Comment:
I think we can either remove the snapshot test, or increase the label so that is sufficiently truncates in order to get the snapshot test to show the tooltip.

Reply:
I'll remove them.